php/libraries/NDB_Page.class.inc

```diff
@@ -590,6 +600,89 @@ class NDB_Page
         return $this->form->createElement('password', $field, $label, $attr);
     }
 
+    /**
+     * An NDB_Page can act as both a middleware, or a handler, depending on the
+     * context. When treated as a middleware, the page is responsible for setting
+     * up any middleware required to access the page. By default, this will
+     * implement the page decoration middlewares which is responsible for adding
+     * the LORIS menu's/headers/footers/etc.
+     * This can be overridden by modules who don't want this behaviour (such
+     * as API endpoints.)
```

Comment:
... endpoint**).**

Reply:
There are multiple endpoints in the API.

Reply:
dot should be after ending parentheses. Fixed the missing s in my comment.

Reply:
that looks weird to me, but most styleguides I can find agree with you, so I moved it outside.